### PR TITLE
Merge 1 8

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -945,6 +945,7 @@
             <configuration>
               <additionalparam>-Xdoclint:none</additionalparam>
               <failOnError>false</failOnError>
+              <additionalparam>-Xdoclint:none</additionalparam>
```
Comment (elfenheart) [inline pom.xml]: duplicate with line 946?

Comment (albertshau) [reply pom.xml]: good catch, guess it was done on both branches

**Comment by bdmogal** [comment]
Everything else seems ok, but https://github.com/caskdata/hydrator-plugins/pull/758 is wrong, since it doesn't update the spec version to 1.5. @ajainarayanan ?

Not sure how we want to handle this.

**Comment by albertshau** [comment]
after discussion, we'll merge this, as the spec stuff is a separate and larger issue

**Pull request merged**